### ISSUE TITLE
Cannot modify links in core code with Whitelabel Plugin Enabled

### ISSUE THREAD

**Issue by craigvc** [piwik/piwik#20408]
## Expected Behavior

Should be able to customise links to point to whitelabel version of pages.

## Current Behavior

Links go to Matomo website

Cannot provide a white label version
## Possible Solution

Allow links currently harcoded in view files to be modified (translation files for multilanguage?)

**Comment by bx80** [comment]
Thanks for the suggestion @craigvc, we'll need to devise a way to make the links modifiable that will work consistently across the codebase.

I've assigned this issue  for prioritization.

**Comment by AltamashShaikh** [comment]
@craigvc Can you specify the usecase for modifying the links ? as the links points to FAQ or guides written by Matomo team.

**Comment by craigvc** [comment]
@AltamashShaikh Yes as we said we are using the whitelabel plugin, so we would want to direct users to our own integration guides etc.

**Comment by AltamashShaikh** [comment]
@craigvc Thanks for the clarification, not sure if this will be easily doable but we have a better idea why this is required now

**Comment by Stan-vw** [comment]
Thanks for your feature request. As we currently see a low amount of demand from users for this feature, we can't prioritise it over the currently planned work. If more users ask for it in the future we'll consider it again. Thanks for your understanding.

**Comment by craigvc** [comment]
It seems like I pay for white label and don’t get it?
